test/selenium/print_test.js

```diff
@@ -11,6 +11,8 @@ var runTest = function(cap, driver, target) {
   driver.findElement(webdriver.By.xpath("//a[contains(text(), 'Grundlagen und Planung')]"));
   // Click on "Drucken"
   driver.findElement(webdriver.By.xpath("//a[@id='printHeading']")).click();
+  // Wait until configuration is loaded
+  driver.findElement(webdriver.By.xpath("//span[@ng-if='printConfigLoaded']"));
```

Comment:
Couldn't we juste test if there is options in  one of the selectbox? 

I'm not a fan of added scope property only for test